.gitignore

```diff
@@ -1,3 +1,5 @@
 # Dev stuff
 __pycache__
 *.sw?
+# Locally generated
+settings
```

Comment:
should be /settings; we only want to ignore the specific settings directory in the root, not all files named "settings"